runtime/compiler/env/J9CPU.hpp

```diff
@@ -45,6 +46,15 @@ class CPU : public OMR::CPUConnector
 public:
 
    J9ProcessorDesc *TO_PORTLIB_getJ9ProcessorDesc();
+
+   // The following method isn't supposed to be on the top level class
+   // However, there exists places where this x86 specific query was called in common code
+   // Its empty definition is here to avoid compilation failures.
+   bool getX86SupportsSSE4_1() { TR_ASSERT(false, "X86 specific query called on non-X86 platform!"); return false; }
```

Comment:
@mstoodle ; Hi Mark, I'm still not sure how to get rid of this guy `getX86SupportsSSE4_1()` on the top level class. Can we just leave this workaround as it is and let the X codegen/optimizer people figure out what to do with it? if not, I'm gonna need help with this one lol.

Reply:
With current code, it is impossible to get rid of this guy; however the place where it is queried is such a hack and we should clean it up. It is straightforward to clean it up but it is beyond the scope of this pull request. Let me create a pull request to remove its caller so that you can safely remove this guy once merged.

Reply:
Once https://github.com/eclipse/openj9/pull/2719 is merged, `getX86SupportsSSE4_1()` can be removed.